### PR TITLE
& fkiraly [ENH] Add skchange change point detectors as placeholder records

### PR DESCRIPTION
**Pull request by ShivamJ07**
Towards https://github.com/sktime/sktime/issues/7351. Adds all current change point detectors from `skchange` as placeholder records to `sktime`, for display in API reference and estimator overview.

Imports for models in this newly added folder will be redirected to skchange (provided that the required version of skchange is installed).

### CODE REVIEW

File: sktime/detection/skchange/moving_window.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""Moving Window detector."""
+
+from sktime.detection.base import BaseDetector
+from sktime.utils.dependencies import _placeholder_record
+
+
+@_placeholder_record(
+    "skchange.change_detectors.moving_window", dependencies="skchange>=0.6.0"
```
Comment (ShivamJ07) [inline sktime/detection/skchange/moving_window.py:8]: Is there any direction on whether explicitly specifying the `skchange` dependency is necessary? If so, what should be the version? How would we handle the case where user has a version mismatch the sktime placeholder is imported instead?

**Review by fkiraly — CHANGES_REQUESTED** [review]
Thanks, looks great!

Some notes:

* also ensure adding the estimator(s) to the API reference, in `docs/api_reference`
* the `dependencies` tag in the `_placeholder_record` needs to be set only when the tag is not set. In this case, I would simply set the tag `python_dependencies="skchange>=0.6.0"`.
* The docstring should be identical with that in `skchange`. I do hope there is a more complete docstring?
    * With one potentila exception: in the summary line, we should also add the qualifier "from `skchange`", and in a second line give the full path in `skchange`. This is read by the doc generator and displayed in the table in the API reference - so those summary positions give credit to `skchange` and its developer(s).
* The `__init__` should also be copied, at least with the full set of arguments, because this is read by the doc generator

Have a look at `HierarchicalProphet` for a no-exceptions example

**Comment by fkiraly** [comment]
Thanks! To expedite this for the presentation, I will take this PR and finish it.

**Comment by ShivamJ07** [comment]
> Thanks! To expedite this for the presentation, I will take this PR and finish it.

Sounds good, let me know if I can be of any help.

**Pull request merged**